nemo/collections/common/parts/preprocessing/collections.py

```diff
@@ -322,12 +330,16 @@ def __init__(self, manifests_files: Union[str, List[str]], is_regression_task=Fa
             audio_files.append(item['audio_file'])
             durations.append(item['duration'])
             if not is_regression_task:
-                labels.append(item['label'])
+                label = item['label']
             else:
-                labels.append(float(item['label']))
+                label = float(item['label'])
 
+            labels.append(item['label'])
```

Comment:
you meant to write `labels.append(label)` here I guess?

Reply:
good catch!!!!